client/src/_Tests_/Questions/QuestionsList.test.jsx

```diff
@@ -5,8 +5,35 @@ import '@testing-library/jest-dom';
 import QuestionsList from '../../components/Questions/QuestionsList';
 
 describe('Question List for indivdual product', () => {
+  const questions = {
```

Comment:
Zach is going to tell you to set up a mock service worker instead of hardcoding mock data